setup_api.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
Don't think this is needed as well

Reply:
If I want to get the api running I needed this script run.